### ISSUE TITLE
Wrong time of the last file updates in the file info page webUI

### ISSUE THREAD

**Issue by ostasevych** [ONLYOFFICE/DocumentServer#2405]
Hi! I've found that the time of the last update is displayed incorrectly on the webUI spreadsheet file info page.
Eg, I have made some changes today at 10:07 EEST, however I see that it displays 13:07.
What may be wrong?

![oo_wrong_date_update](https://github.com/ONLYOFFICE/onlyoffice-nextcloud/assets/5199030/3000af34-7376-469c-8762-db2d52a436c2)

OnlyOffice last version in docker.

**Comment by rsnegirev** [comment]
Hi @ostasevych 

Sorry for a long answer.

I found that the problem is in the Libreoffice Calc editor.

Look, I have GMT +4:00 Tbilisi and I just created a file in Libre, saved it as XLSX and opened it in MS Excel and OnlyOffice editor, and this is what I got:
![image](https://github.com/ONLYOFFICE/DocumentServer/assets/65603548/ed2134b7-af9a-43a1-a3c1-06cdf7cb1a7a)
![image](https://github.com/ONLYOFFICE/DocumentServer/assets/65603548/702416dc-e707-4045-bf5e-04a626d92466)

Probably Libreoffice in this case doesn't use a local time, but from a server somewhere in China.
